testsuite/src/main/java/io/netty/testsuite/transport/socket/DatagramUnicastTest.java

```diff
@@ -397,17 +356,51 @@ private static ChannelFuture write(Channel cc, ByteBuf buf, WrapType wrapType) {
         }
     }
 
-    @SuppressWarnings("deprecation")
-    private Channel setupServerChannel(Bootstrap sb, final byte[] bytes, final SocketAddress sender,
+    protected boolean isConnected(Channel channel) {
+        return ((DatagramChannel) channel).isConnected();
+    }
+
+    protected Channel setupClientChannel(Bootstrap cb, final byte[] bytes, final CountDownLatch latch,
```

Comment:
Likewise I think this could be pushed down?

Reply:
done